src/simuq/braket/braket_provider.py

```diff
@@ -87,13 +91,54 @@ def compile(
             self.prog = self.ahs_prog
             self.layout = layout
 
+        elif self.provider == "ionq":
+            if device == "Harmony":
+                nsite = 11
+            elif device == "Aria-1":
+                nsite = 23
+            elif device == "Aria-2":
+                nsite = 23
+            elif isinstance(device, int):
```

Comment:
What does this mean again?

Reply:
Copied from the IonQ direct access implementation; I presume this just sets the number of sites.